src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Lookup.cs

```diff
@@ -101,25 +129,54 @@ public override bool IsServerDelegatable(CallNode callNode, TexlBinding binding)
                 metadata = dataSource.DelegationMetadata.FilterDelegationMetadata;
             }
 
-            var args = callNode.Args.Children.VerifyValue();
-            if (args.Count > 2 && binding.IsDelegatable(args[2]))
-            {
-                SuggestDelegationHint(args[2], binding);
-                return false;
-            }
+            return true;
+        }
 
-            if (args.Count < 2)
+        public override bool IsEcsExcemptedLambda(int index)
```

Comment:
>Excempted [](http://example.com/codeflow?start=34&length=9)

Typo - `Exempted`. Since this is defined in the base class it's ok to leave it like that, but can you at least fix the comment?

Reply:
This looks like a merge artifact